api/swagger.yaml

```
@@ -210,6 +210,32 @@ definitions:
       PathInContainer: "/dev/deviceName"
       CgroupPermissions: "mrw"
 
+  DeviceRequest:
+    type: "object"
+    description: "A request for devices to be sent to device drivers"
+    properties:
+      Driver:
+        type: "string"
+      Count:
+        type: "string"
+      DeviceIDs:
+        type: "array"
+        items:
+          type: "string"
+      Capabilities:
+        type: "array"
+        items:
+          type: "array"
+          items:
+            type: "string"
+      Options:
```

Perhaps this should be `DriverOpts` (which is what we use elsewhere)
```suggestion
      DriverOpts:
```

---

Hm, actually, looks like we use `Options` in another location (for IPAM, LogDriver))

---

Let's keep it as `Options`